src/dbt/kipptaf/models/extracts/google/sheets/rpt_gsheets__stipends_app_roster.sql

```diff
@@ -0,0 +1,101 @@
+with
+    approval_loops as (
+        select
+            home_work_location_name as location,
+            business_unit_home_name as region,
+            department_home_name as department,
+            preferred_name_lastfirst as first_approver_name,
+            user_principal_name as first_approver_email,
+            google_email as first_approver_google,
+            report_to_preferred_name_lastfirst as second_approver_name,
+            report_to_mail as second_approver_email,
+            case
+                when business_unit_home_name not like '%Miami%'
+                then concat(report_to_sam_account_name, '@apps.teamschools.org')
+                when business_unit_home_name like '%Miami%'
+                then concat(report_to_sam_account_name, '@kippmiami.org')
+            end as second_approver_google,
+            employee_number as first_approver_employee_number,
+            report_to_employee_number as second_approver_employee_number,
+            case
+                when department_home_name = 'Operations'
+                then 'Operations'
+                when department_home_name = 'School Leadership'
+                then 'Instructional'
+                when business_unit_home_name like '%Family%'
+                then 'CMO'
+                else 'Regional'
+            end as route
+        from {{ ref("base_people__staff_roster") }}
+        where
+            job_title in (
+                'Director School Operations',
+                'Director Campus Operations',
+                'Director',
+                'School Leader',
+                'School Leader in Residence'
+            )
+            and assignment_status != 'Terminated'
+    ),
+
+    roster as (
+        select
+            sr.employee_number,
```

Comment:
group cols by source table and order from simple (no changes) to complex (case statements)